createPrsToUpdateReleaseDrafterAndPublisherWorkflows.sh

```diff
@@ -0,0 +1,179 @@
+#!/bin/bash
+
+# Release Drafter Workflow Creator CLI
+# ===================================
+# This script creates pull requests to add a Release Drafter workflow to each repository
+# in the axonivy-market GitHub Organization.
+# Using https://cli.github.com/
+
+org="axonivy-market"
+
+ignored_repos=(
+  "market-up2date-keeper"
+  "market.axonivy.com"
+  "market-monitor"
+  "market"
+  "demo-projects"
+)
```

Comment:
I've made this collection of repos, which we use more or less in every script of this repo, a re-usable script. I think it should also fit into this one: see https://github.com/axonivy-market/market-up2date-keeper/blob/master/repo-collector.sh
you may simply include it like this:
```shell
DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
. ${DIR}/repo-collector.sh
```